www/js/app.js

```diff
@@ -414,9 +417,8 @@ $('#btnHome').on('click', function () {
     }
     // Use a timeout of 400ms because uiUtil.applyAnimationToSection uses a timeout of 300ms
     setTimeout(resizeIFrame, 400);
-    return false;
 });
-$('#btnConfigure').on('click', function () {
+document.getElementById('btnConfigure').addEventListener('click', function () {
```

Comment:
Like with `btnHome` you here need to add an `event.preventDefault();` and add `event` in the function parameters.